domain/relation/state/relation.go

```diff
@@ -38,6 +37,32 @@ func NewState(factory database.TxnRunnerFactory, clock clock.Clock, logger logge
 	}
 }
 
+// GetAllRelationDetails return all uuid of all relation for the current model.
```

Comment:
todo: update comment to match the method. I'd be specific has there will be overlap with other methods getting "details", we must differentiate.